tests/test_armv7cpu.py

```diff
@@ -1695,3 +1696,35 @@ def test_arm_save_restore_cpu(self):
         dumped_s = pickle.dumps(self.cpu)
         self.cpu = pickle.loads(dumped_s)
 
+    def test_symbolic_conditional(self):
+        asm = \
+        """
+          tst r0, r0
+          beq label
+          bne label
+        label:
+          nop
+        """
```

Comment:
Can we rewrite the code so the addresses are more explicit?

e.g.
```python
# in thumb mode each instruction has size of 2 bytes
# (the code starts at 0x1004)
asm = '\n'.join(
    '  tst r0, r0',  # 0x1004
    '  beq label'  # 0x1006
    '  bne label'  # 0x1008
    'label:'
    '  nop'  # 0x100a
)
```

Reply:
Done, see my latest commit. :smile: